lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -307,6 +373,11 @@ defmodule ExUnit.DocTestTest do
   end
 
   test "doctest failures" do
+
```

Comment:
Let's remove newlines around this comment.
We also don't use backticks in comments.